packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -1371,7 +1372,23 @@ class RawScrollbarState<T extends RawScrollbar> extends State<T> with TickerProv
     if (scrollOffsetGlobal != position.pixels) {
       // Ensure we don't drag into overscroll if the physics do not allow it.
       final double physicsAdjustment = position.physics.applyBoundaryConditions(position, scrollOffsetGlobal);
-      position.jumpTo(scrollOffsetGlobal - physicsAdjustment);
+      double newPosition = scrollOffsetGlobal - physicsAdjustment;
+
+      // The physics may allow overscroll when actually *scrolling*, but
+      // dragging on the scrollbar does not always allow us to enter overscroll.
+      switch(ScrollConfiguration.of(context).getPlatform(context)) {
+        case TargetPlatform.android:
+        case TargetPlatform.fuchsia:
+        case TargetPlatform.linux:
+        case TargetPlatform.macOS:
+        case TargetPlatform.windows:
+          newPosition = newPosition.clamp(0.0, position.maxScrollExtent);
```

Comment:
We can use the 'Bounce physics' on non-iOS platform, why not allow over scrolling in this case?

Reply:
Oh, I see，to be consistent with the behavior of the platform.

Reply:
How about let drag overscroll on the mobile platform, you know that some developers want to keep the ios behavior on Android phones uses bounce physics.

Reply:
I think that sounds good. I'll update. Thanks! :)